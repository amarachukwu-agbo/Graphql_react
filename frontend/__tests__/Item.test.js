import { shallow } from 'enzyme';
import Item from '../components/Item';

const props = {
  item: {
    image: 'https://res.cloudinary.com/amara/sic-fits/dog.png',
    title: 'A chihuahua dog',
    id: 'ctruinb198338',
    price: 500000,
    description: 'A beautiful chihuahua dog'
  }
}
describe('<Item />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Item {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });
});