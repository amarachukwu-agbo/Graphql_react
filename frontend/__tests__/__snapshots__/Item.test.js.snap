// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> matches snapshot 1`] = `
<ItemStyles__Item>
  <img
    alt="A chihuahua dog"
    src="https://res.cloudinary.com/amara/sic-fits/dog.png"
  />
  <Title>
    <Link
      href={
        Object {
          "pathname": "/item",
          "query": Object {
            "id": "ctruinb198338",
          },
        }
      }
    >
      <a>
        A chihuahua dog
      </a>
    </Link>
  </Title>
  <PriceTag>
    $5,000
  </PriceTag>
  <p>
    A beautiful chihuahua dog
  </p>
  <div
    className="buttonList"
  >
    <Link
      href={
        Object {
          "pathname": "update",
          "query": Object {
            "id": "ctruinb198338",
          },
        }
      }
    >
      <a>
        Edit
      </a>
    </Link>
    <AddToCart
      id="ctruinb198338"
    />
    <DeleteItem
      id="ctruinb198338"
    >
      Delete
    </DeleteItem>
  </div>
</ItemStyles__Item>
`;
